update for firefox 47+ and adapter 2.0

fippo opened the pull request:
@ggarber @aullman can you merge this (assuming travis passes)

fippo committed:
update for firefox 47+ and adapter 2.0

aullman commented:
Great! Travis doesn't seem to be setup for this repo though. I just turned it on and I'm going to close and open this PR again to trigger a Travis build.

Commit:
Update node version to 6

It seems like selenium webdriver needs a newer version 
https://travis-ci.org/opentok/upgrade-test/jobs/194710263

aullman commented:
@fippo this is still failing. I'm not sure why.
